tests/installation/bootloader_uefi.pm

```diff
@@ -54,10 +56,14 @@ sub run {
         sleep 5;
     }
     if (get_var("ZDUP")) {
-        # uefi bootloader has no "boot from harddisk" option. So we
-        # have to just reboot here
-        eject_cd;
-        power('reset');
+        # 'eject_cd' is broken ATM (at least on aarch64), so select HDD from menu - poo#47303
+        # Check we are booting the ISO
+        assert_screen 'inst-bootmenu';
+        # Select boot from HDD
+        send_key_until_needlematch 'inst-bootmenu-boot-harddisk', 'up';
+        send_key 'ret';
+        # use firmware boot manager of aarch64 to boot HDD
+        $self->handle_uefi_boot_disk_workaround if (check_var('ARCH', 'aarch64'));
```

Comment:
Should we apply this workaround for arm only and keep old behavior for qemu on x86_64? For me it's hard to say if it will work there.

Reply:
We do not have any UEFI ZDUP test on o3 and I doubt the `eject_cd` works anymore on x86_64 too, with changes about CD handling in os-autoinst (being a block device now).

Reply:
Let me check osd.

Reply:
Alright, I was not able to find UEFI ZDUP on OSD, to it's safe.

Reply:
ok, thanks.